scripts/kubernetes-all-test.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up some stuff"
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+mkdir bin artifacts
+# Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
+# for eg go get ginkgo later on will produce ginkgo binary in GOBIN
+export GOBIN="`pwd`/bin"
+
+# Set kubeconfig to current dir. This ensures no clashes with other test runs
+export KUBECONFIG="`pwd`/config"
+export ARTIFACTS_DIR="`pwd`/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
+
+# This si one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+
+# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
+export PATH=$PATH:$GOBIN
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avrf ./odo $GOBIN/
+shout "getting ginkgo"
+make goget-ginkgo
+
+shout "Start minikube"
+# Delete minikube instance, if in anycase already exists
+minikube delete
+minikube start --vm-driver=docker --container-runtime=docker
+set +x
+# Get kubectl cluster info
+kubectl cluster-info
+
+set -x
+# Set kubernetes env var as true, to distinguish the platform inside the tests
+export KUBERNETES=true
+
+# Integration tests
+shout "Running integration Tests"
+make test-cmd-project
```

Comment:
Is this really going to run all the tests that we have?

Reply:
We should probably have a place that executes all the scripts, I already made the same command at https://github.com/openshift/odo/pull/4406/files#r588262224

Reply:
@rnapoles-rh I think you were looking into consolidating scripts at some point of time, correct me if I am wrong. Are you still looking into it. If yes could you please give some update on it, so that this pr merge gets smooth.

Reply:
@prietyc123 yes, in https://github.com/openshift/odo/pull/4406 I consolidated your scripts with mine creating the following files:
minikuke-minishift-setup-env.sh
minikube-minishift-all-tests.sh
minici-presubmit-all-tests.sh
PTAL